query/stdlib/experimental/to.go

```diff
@@ -5,24 +5,23 @@ import (
 	"errors"
 	"fmt"
 
-	platform2 "github.com/influxdata/influxdb/v2/kit/platform"
-
 	"github.com/influxdata/flux"
 	"github.com/influxdata/flux/codes"
 	"github.com/influxdata/flux/execute"
 	"github.com/influxdata/flux/plan"
 	"github.com/influxdata/flux/runtime"
 	"github.com/influxdata/flux/semantic"
-	"github.com/influxdata/flux/stdlib/experimental"
+	_ "github.com/influxdata/flux/stdlib/experimental"
```

Comment:
Is this import still needed? I'd expect any initialization side-effects to be covered by [this import](https://github.com/influxdata/influxdb/blob/master/fluxinit/init.go#L12).

Reply:
It's not needed for the main function but the `query/stdlib/experimental` doesn't seem to use that import so this import just prevents the initialization from happening in the wrong order. Since this package requires the experimental package to be imported before the `init()` is executed, might as well make it explicit rather than relying on the consumer to import it correctly.